cmd/asset-syncer/utils.go

```diff
@@ -217,7 +224,7 @@ func filterCharts(charts []models.Chart, filterRule *apprepov1alpha1.FilterRuleS
 }
 
 // Charts retrieve the list of charts exposed in the repo
-func (r *HelmRepo) Charts() ([]models.Chart, error) {
+func (r *HelmRepo) Charts(shallow bool) ([]models.Chart, error) {
```

Comment:
Similar here, I think `latest_charts_only` might better indicate what actually happens, here and below.